backend/tests/unit/dal/Libraries/Ches/ChesExceptionTest.cs

```diff
@@ -0,0 +1,149 @@
+using FluentAssertions;
+using Pims.Core.Exceptions;
+using Pims.Core.Http;
+using Pims.Core.Test;
+using Pims.Ches.Models;
+using Moq;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using Xunit;
+using Pims.Core.Http.Models;
+using Microsoft.Extensions.Options;
+using Pims.Ches.Configuration;
+using Pims.Ches;
+using System.Linq;
+using System.Net.Http;
+
+namespace Pims.Dal.Test.Libraries.Ches
+{
+    [Trait("category", "unit")]
+    [Trait("category", "ches")]
+    [Trait("group", "notification")]
+    [ExcludeFromCodeCoverage]
+    public class ChesExceptionTest
+    {
+
+        #region Tests
+        #region ChesException Constructors
+        [Fact]
+        public void ChesExceptionClientModel_Success()
+        {
+            // Arrange
+            var message = "test2";
+            var status = HttpStatusCode.OK;
+            var helper = new TestHelper();
+
+            var user = PrincipalHelper.CreateForPermission();
+
+            var options = Options.Create(new ChesOptions()
+            {
+                AuthUrl = "https:/test.com",
+                EmailEnabled = true,
+                EmailAuthorized = true,
+                BccUser = true
+            });
+            var service = helper.Create<ChesService>(options, user);
+
+            var token = new TokenModel()
+            {
+                AccessToken = "test"
+            };
+
+            var exception = new HttpClientRequestException(message, status);
+            var client = helper.GetService<Mock<IHttpRequestClient>>().Object;
+            var model = new ErrorResponseModel();
+
+            // Act
+            var chesException = new ChesException(exception, client, model);
+
+            // Assert
+            Assert.NotNull(exception);
+            Assert.NotNull(client);
+            Assert.NotNull(model);
+            chesException.Message.Should().Be(message + System.Environment.NewLine);
```

Comment:
Probably should find out why it has an extra `NewLine` at the end of the string.  It's probably not ideal.

Reply:
There is a `NewLine` in the constructor:

`public ChesException(HttpClientRequestException exception, IHttpRequestClient client, ErrorResponseModel model)
            : this($"{exception.Message}{Environment.NewLine}", exception, exception.StatusCode)`

where it concatenates the new line right after the exception.Message.

Reply:
Ah leave it for now.  Looks a little messy, but it isn't critical.